using-valtimo/upload/upload-to-documenten-api-with-metadata.md

```diff
@@ -2,12 +2,35 @@
 
 The Documenten API is an API for storage and disclosure of documents and associated metadata. Valtimo supports uploading
 files to the Documenten API with the option to input metadata through a dedicated tab on the detail page of a case
-instance.
+instance, or through a custom Form.IO component.
 
-## Configuring the front-end implementation
+## Configuring the Documenten API tab
 
 To enable the dedicated Documenten API tab in an implementation, set `uploadProvider` in the implementation's
 environment file to `UploadProvider.DOCUMENTEN_API`. The documents tab on a case detail page will now show the
 Documenten API uploader.
 
+## Configuring the Documenten API Form.IO upload component
 
+In order to make the Documenten API File Upload component available in Form.IO forms in your implementation, import
+`registerDocumentenApiFormioUploadComponent` from `@valtimo/components` in your app module and move it into the
+`AppModule` constructor like so:
+
+#### **`app.module.ts`**
+  ```typescript
+...
+import {
+    ...
+    registerDocumentenApiFormioUploadComponent,
+} from '@valtimo/components';
+
+...
+export class AppModule {
+    constructor(injector: Injector) {
+        ...
+        registerDocumentenApiFormioUploadComponent(injector);
+    }
+}
+  ```
+
+Afterwards, the component is available under the advanced section in the form builder.
```

Comment:
Afterwards goes at the end of the sentence.

Reply:
Using afterwards at the beginning of a sentence like this seems to be grammatically correct: https://sentence.yourdictionary.com/afterwards. So I don't think we should comment on these kinds of stylistic choices as long as the meaning is clear.

Reply:
Generally people don't seem to be content with yourdictionary. Not sure if we should use this to supplement our arguments. Either way though, this is a nitpicky thing. Everyone will understand it, which I guess is good enough.